ci/terraform-import.yml

```diff
@@ -0,0 +1,58 @@
+env:
+  GIT_AUTHOR_NAME: Buildkite
+  GIT_AUTHOR_EMAIL: ci@buildkite
+  GIT_COMMITTER_NAME: Buildkite
+  GIT_COMMITTER_EMAIL: ci@buildkite
+  NIX_PATH: nixpkgs=channel:nixos-unstable-small
+  NIXPKGS_ALLOW_UNFREE: 1
```

Comment:
/me wonders what this is for

Reply:
For what it is worth, I'd rather totally avoid setting the NIX_PATH (environment variable) and this NIXPKGS_ALLOW_UNFREE variable anywhere.